.github/ISSUE_TEMPLATE/docs-request.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Documentation Request
+about: General request for the Observability Documentation Team.
+title: "[Request]"
+labels: Team:Docs
+assignees: ''
+projects: 513
```

Comment:
This will bypass the Fleet/Agent project board unless we tell folks to change the project ID for Fleet or Elastic Agent-related issues. But I guess we can just add the project manually when we triage issues?

TBH, I'm hoping we can sunset the Fleet/Agent project board after we have the initial docs all written. At that point, we shouldn't have so many issues that we need a separate board. That's the hope/dream.

Reply:
Is there any automation in place to add to the Fleet board?